rct229/rulesets/ashrae9012019/section19/section19rule21.py

```diff
@@ -0,0 +1,498 @@
+                            "hvac_fan_sys_exhaust_sum"
+                        ],
+                        hvac_systems_and_assoc_zones_largest_exhaust_source[
+                            "maximum_zone_exhaust"
+                        ],
+                    )
+                    < 0.75 * max_outdoor_airflow_b
+                )
+
+                exception_7_applies = (
+                    not ER_modeled_p and serves_zones_that_have_dehumid_heat_recovery
+                )
+
+                return {
+                    "climate_zone": climate_zone,
+                    "ER_not_req_for_heating_sys": ER_not_req_for_heating_sys,
+                    "serves_zones_with_systems_likely_exhausting_toxic_etc": serves_zones_with_systems_likely_exhausting_toxic_etc,
+                    "serves_kitchen_space": serves_kitchen_space,
+                    "all_lighting_space_types_defined": all_lighting_space_types_defined,
+                    "all_ventilation_space_types_defined": all_ventilation_space_types_defined,
+                    "serves_zones_that_have_dehumid_heat_recovery": serves_zones_that_have_dehumid_heat_recovery,
+                    "sys_type_heating_only": sys_type_heating_only,
+                    "supply_airflow_b": CalcQ("air_flow_rate", supply_airflow_b),
+                    "min_outdoor_airflow_b": CalcQ(
+                        "air_flow_rate", min_outdoor_airflow_b
+                    ),
+                    "OA_fraction_b": OA_fraction_b,
+                    "ER_modeled_b": ER_modeled_b,
+                    "ER_modeled_p": ER_modeled_p,
+                    "max_outdoor_airflow_b": max_outdoor_airflow_b,
+                    "hvac_systems_and_assoc_zones_largest_exhaust_source": hvac_systems_and_assoc_zones_largest_exhaust_source,
+                    "exception_1_applies": exception_1_applies,
+                    "exception_2_applies": exception_2_applies,
+                    "exception_6_applies": exception_6_applies,
+                    "exception_7_applies": exception_7_applies,
+                }
+
+            def manual_check_required(self, context, calc_vals=None, data=None):
+                OA_fraction_b = calc_vals["OA_fraction_b"]
+                supply_airflow_b = calc_vals["supply_airflow_b"]
+                ER_modeled_b = calc_vals["ER_modeled_b"]
+                ER_modeled_p = calc_vals["ER_modeled_p"]
+                serves_kitchen_space = calc_vals["serves_kitchen_space"]
+                max_outdoor_airflow_b = calc_vals["max_outdoor_airflow_b"]
+                hvac_systems_and_assoc_zones_largest_exhaust_source = calc_vals[
+                    "hvac_systems_and_assoc_zones_largest_exhaust_source"
+                ]
+
+                return (
+                    # Case 7
+                    (
+                        OA_fraction_b >= OA_fraction_b_70
+                        and supply_airflow_b >= SUPPLY_AIRFLOW_5000CFM
+                    )
```

Comment:
You can move the `not ER_modeled_b` to the first block

Reply:
Addressed.